recipes/texture-atlas/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: texture-atlas
+  version: {{ environ['GIT_DESCRIBE_TAG'] ~ "." ~ environ['GIT_DESCRIBE_NUMBER'] ~ "." ~ environ['GIT_DESCRIBE_HASH'] }}
+
+source:
+#  git_rev: {{ sha }}
+  git_branch: add-findproj4
+  git_url: git@gitlab.kitware.com:danlipsa/core3d-keu.git
+  patches:
+    - gdal-headers.patch
+
+requirements:
+  build:
+    - vtk
+    - eigen 3.3.3.*
+    - opencv 3.3.1.*
+    - libgdal 2.2.3.*
+  run:
+    - vtk
+    - eigen 3.3.3.*
+    - opencv 3.3.1.*
+    - libgdal 2.2.3.*
+
```

Comment:
There could be a `test` section here as a sanity check that the executables run. In the simplest case this is just:
```yaml
test:
  commands:
    - run_texture_mapping
    - run_only_atlases_fusion
    - run_only_obj_generation
```
What do you think?

Reply:
Sounds good. I added that.